### PR TITLE
https://github.com/DominoKit/domino-ui/issues/104

### PR DESCRIPTION
**Pull request by schube**

### CODE REVIEW

File: src/main/java/org/dominokit/domino/ui/forms/ValueBox.java [hunk 1]
```diff
@@ -255,13 +255,15 @@ public T setRightAddon(Element rightAddon) {
     public T removeRightAddon() {
         if (nonNull(rightAddon)) {
             rightAddonContainer.removeChild(rightAddon);
```
Comment (vegegoku) [inline src/main/java/org/dominokit/domino/ui/forms/ValueBox.java]: I think the solution the issue should be by replacing `rightAddonContainer.removeChild(rightAddon);` with `rightAddon.remove()`

File: src/main/java/org/dominokit/domino/ui/forms/ValueBox.java [hunk 2]
```diff
@@ -255,13 +255,15 @@ public T setRightAddon(Element rightAddon) {
...
+            rightAddon = null;
         }
         return (T) this;
     }
 
     public T removeLeftAddon() {
         if (nonNull(leftAddon)) {
             leftAddonContainer.removeChild(leftAddon);
```
Comment (vegegoku) [inline src/main/java/org/dominokit/domino/ui/forms/ValueBox.java]: same as above

Comment (schube) [reply src/main/java/org/dominokit/domino/ui/forms/ValueBox.java]: Sure, whatever fixes the problem. You are the expert!

**Comment by vegegoku** [comment]
this is should work now

**Pull request closed without merging**